### PR TITLE
Feature/tapasrecord

### PR DESCRIPTION
**Pull request by LindaJeanne**
This fixes the issue that was still in the pull request from yesterday -- adds the module to the buildscript that was needed for the changes.

### CODE REVIEW

File: buildtapas.make [hunk 1]
```diff
@@ -56,7 +57,7 @@ projects[diff][subdir] = develop
 projects[tapas-modules][type] = module
 projects[tapas-modules][download][type] = git
 projects[tapas-modules][download][url] = git://github.com/NEU-DSG/tapas-modules
-projects[tapas-modules][download][branch] = develop
+projects[tapas-modules][download][branch] = feature/tapasrecord
```
Comment (jbuckle) [inline buildtapas.make]: Shouldn't this be develop again?

Comment (LindaJeanne) [inline buildtapas.make]: Yup. That's the weird thing about merges. I'll update.

**Comment by LindaJeanne** [comment]
Updated to pull from the correct paths in the repositories for merge.

File: buildtapas.make [hunk 2]
```diff
@@ -56,7 +57,7 @@ projects[diff][subdir] = develop
 projects[tapas-modules][type] = module
 projects[tapas-modules][download][type] = git
 projects[tapas-modules][download][url] = git://github.com/NEU-DSG/tapas-modules
-projects[tapas-modules][download][branch] = develop
+projects[tapas-modules][download][branch] = feature/develop
```
Comment (jbuckle) [inline buildtapas.make]: @lindajeanne This has a typo in it.  The branch should be develop, not feature/develop.

**Comment by LindaJeanne** [comment]
typo fixed.

**Comment by jbuckle** [comment]
Alright, this looks good to me now.

**Pull request merged**